examples/pseudo/config.json

```diff
@@ -0,0 +1,34 @@
+{
+
+    "shared"            : "libdice.so",
+    "database_filename" : "pseudo.sqlite",
+
+    "parameters" : [
+        {"name"       : "number of dice",
+         "short_name" : "ndice",
+         "dist_type"  : "PSEUDO",
+         "num_type"   : "INT",
+         "par1"       : 1,
+         "par2"       : 5
```

Comment:
It would be nice to show in at least one example that step is an available argument

Reply:
plan to resolve this via #46 discussion